src/Discovery/Discovery.tsx

```diff
@@ -322,7 +333,7 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
       // in the table and the study description.
       // ---
       const searchableFields = selectedSearchableTextFields;
-      if (searchableFields.length > 0) {
+      if (searchableFields.length > 0 || searchMode === SearchMode.RESTRICTED) {
```

Comment:
Question: do we still need to check for both? Or can we just check for the ```searchMode === SearchMode.RESTRICTED```

Reply:
we need to check for both since `searchableFields.length > 0` is for the default case for envs that doesn't use restricted search mode